src/index.js

```diff
@@ -102,22 +88,35 @@ export default class Marker extends Component {
   // render
 
   render () {
-    const { left, top, onClick } = this.props
+    const { left, top, onClick, markerColor, markerColorHover, markerSize } = this.props
+    const { hover } = this.state
+    const svgSize = {width: markerSize, height: markerSize}
```

Comment:
I don't understand this `svgSize`. Where is it used?

Reply:
I've fixed that.